ConfigSpace/configuration_space.py

```diff
@@ -396,16 +401,16 @@ def get_conditions(self):
 
         return conditions
 
-    def get_children_of(self, name):
+    def get_children_of(self, name: Hyperparameter) -> List[Hyperparameter]:
         conditions = self.get_child_conditions_of(name)
-        parents = []
+        parents = [] # type: List[Hyperparameter]
         for condition in conditions:
             parents.extend(condition.get_children())
         return parents
 
-    def get_child_conditions_of(self, name):
+    def get_child_conditions_of(self, name: Union[str, Hyperparameter]) -> List[AbstractCondition]:
         if isinstance(name, Hyperparameter):
-            name = name.name
+            name = name.name  # type: ignore
```

Comment:
i thought so too but mypy gives following error which doesnt actually make any sense
**configuration_space.py:413: error: Some element of union has no attribute "name"**